sale_stock_ux/models/sale_order_line.py

```diff
@@ -123,3 +132,91 @@ def _onchange_product_uom_qty(self):
             self.product_uom_qty = self._origin.product_uom_qty
             return {'warning': warning_mess}
         return {}
+
+    @api.depends('qty_delivered', 'qty_returned')
+    def _compute_all_qty_delivered(self):
```

Comment:
`_compute_all_qty_delivered` is defined twice in the same file, I think this can be merged